RichText-component/src/main/java/com/celements/rteConfig/RTEConfigScriptService.java

```diff
@@ -61,6 +62,9 @@ public class RTEConfigScriptService implements ScriptService {
   @Requirement
   private IDefaultEmptyDocStrategyRole defaultEmptyDocStrategyRole;
 
+  @Inject
+  private RteConfigRole defaultRteConfig;
```

Comment:
if its not too much work please also use constructor injection here.

Reply:
I tried already. Unfortunately, a refactoring in Spring-Component does not work, because the current IDefaultEmptyDocStrategyRole property expects to get com.celements.emptycheck.internal.DefaultEmptyDocStrategy injected. Yet,  DefaultEmptyDocStrategy implements two interfaces which leads to two beans fitting the injection criterias.

Reply:
Thanks for the info. I documented this on CELDEV-1225